brand.php

```diff
@@ -0,0 +1,98 @@
+    <p>The logotype should always be used under the following guidelines:</p>
+    <ul>
+        <li>Do not attempt to recreate the logotype. It is a meticulously-designed brand mark, not simply "elementary" written in a specific font.</li>
+        <li>Do not use the logotype at small sizes; if it is not clear, use the logomark instead.</li>
+    </ul>
+
+    <h3>Logomark</h3>
+    <p>The "e" logomark is to be used to refer to elementary the company when space is constrained or a square ratio is required.</p>
+
+    <h2>Color</h2>
+    <p>elementary employs the use of color combined with our name and marks to establish our brand. We use the following palette:</p>
+    <div class="palette">
+        <dl class="blue">
+            <dt>Blueberry</dt>
+            <dd class="hex">3892e0</dd>
+        </dl>
+        <dl class="red">
+            <dt>Strawberry</dt>
+            <dd class="hex">da4d45</dd>
+        </dl>
+        <dl class="orange">
+            <dt >Orange</dt>
+            <dd class="hex">f37329</dd>
+        </dl>
+        <dl class="yellow">
+            <dt>Banana</dt>
+            <dd class="hex">fbd25d</dd>
+        </dl>
+        <dl class="green">
+            <dt>Lime</dt>
+            <dd class="hex">93d844</dd>
+        </dl>
+        <dl class="purple">
+            <dt>Grape</dt>
+            <dd class="hex">8a4ebf</dd>
+        </dl>
+        <dl class="white">
+            <dt>White</dt>
+            <dd class="hex">ffffff</dd>
+        </dl>
+        <dl class="black">
+            <dt>Black</dt>
+            <dd class="hex">333333</dd>
+        </dl>
+    </div>
+
+    <h2>Fonts</h2>
+    <p>For web and print, we use Raleway for headings and Open Sans for body copy. For code blocks, we use Roboto Mono.</p>
+
+    <h2>Third Parties</h2>
+    <p>We encourage third party developers creating products for elementary OS to adopt certain elements of the elementary brand to achieve consistency:</p>
+    <ul>
+        <li>Color</li>
+        <li>Fonts</li>
+        <li>Voice/tone</li>
+    </ul>
+    <p>However, to avoid user confusion, we do restrict the usage of the elementary name and marks:</p>
+    <ul>
+        <li>You are encouraged to say that your app or service is designed for elementary OS, but please don't use the elementary name or marks as part of the name of your company, application, product, or service, or in any logo you create.</li>
+        <li>Only use the elementary name or marks to refer to elementary LLC. or its products (i.e. elementary OS).</li>
+    </ul>
+
+    <h2>Merchandise</h2>
+    <p>We do not typically allow our branding (including our name or brand marks) to be used on third-party merchandise.</p>
+
+    <h2>More Info</h2>
+    <p>For further info regarding the use of the elementary name, branding, and trademarks, please email brand@elementary.io.</p>
```

Comment:
Include an actual link for the email? Could lead to more spam, but it's much more convenient for the reader.